Backend/sprintone/src/test/java/com/project/OmegaApplicationTests.java

```diff
@@ -0,0 +1,17 @@
+package com.project;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class OmegaApplicationTests {
+
+    @Test
+    public void contextLoads() {
+    }
```

Comment:
Where are your tests? So many classes, and none of them are tested...

Reply:
The test folder within the project should have all the test cases. The Contextloads() is meant to work as runners. Any listeners, when included, should have contextloads.